tox.ini

```diff
@@ -3,30 +3,33 @@ envlist =
     py{27,py,34,35,36,37}{,-subunit},coverage,docs
 
 [testenv]
-deps =
-    .[test]
-    subunit: .[subunit]
+extras =
+    test
+    subunit: subunit
```

Comment:
Hum, I don't know what that syntax does.

Reply:
https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings